### ISSUE TITLE
[FEATURE] Chromecast support

### ISSUE THREAD

**Issue by AnonymusRaccoon** [AnonymusRaccoon/Kyoo#89]

**Comment by mscheltienne** [comment]
The lack of chromecast support is blocking adoption. I am looking today to move away from Plex; and casting is the number one feature I need. Hope it gets backup in the priority list, I would love to adopt Kyoo!

**Comment by zoriya** [comment]
yes, chromecast is an important part but i've been pretty busy with v5 & #763 recently. I do plan on doing that soonish, but anyone is free to step-up and implement it, i'd love to help anyone willing to contribute

**Comment by zoriya** [comment]
now available on master, next release very soon
